projects/igniteui-angular/schematics/ng-add/index.spec.ts

```diff
@@ -19,16 +40,25 @@ describe('schematics', () => {
   };
 
   const pkgJsonConfig = {
-    dependencies: null,
-    devDependencies: null
+    dependencies: {},
+    devDependencies: {}
   };
 
   beforeEach(() => {
     tree = new UnitTestTree(new EmptyTree());
     tree.create('/angular.json', JSON.stringify(ngJsonConfig));
     tree.create('/package.json', JSON.stringify(pkgJsonConfig));
+    tree.create('src/polyfills.ts', polyfills);
+    populatePackageJson();
   });
 
+  function populatePackageJson() {
+    const pkgJson = JSON.parse(tree.read('/package.json').toString());
+    pkgJson.dependencies['@angular/common'] = '^7.0.3';
```

Comment:
Probably not a great idea to hardcode those versions (you can grab them from the current project dependencies).
Also might want to have one test with a lower versions to verify the warning.